pkgs/servers/homepage-dashboard/default.nix

```diff
@@ -13,13 +13,13 @@
 
 buildNpmPackage rec {
   pname = "homepage-dashboard";
-  version = "0.8.4";
+  version = "0.8.6";
```

Comment:
sorry I wasn't clear enough, commit message upgrades from 0.8.3 not 0.8.4

Reply:
oops! fixed! thank you!